app_flutter/test/state/flutter_build_test.dart

```diff
@@ -210,40 +212,28 @@ void main() {
 
       buildState.dispose();
     });
-
-    test('auth functions call auth service', () async {
-      final MockGoogleSignInService mockSignInService = MockGoogleSignInService();
-      buildState = FlutterBuildState(authServiceValue: mockSignInService);
-
-      verifyNever(mockSignInService.signIn());
-      verifyNever(mockSignInService.signOut());
-
-      await buildState.signIn();
-      verify(mockSignInService.signIn()).called(1);
-      verifyNever(mockSignInService.signOut());
-
-      await buildState.signOut();
-      verify(mockSignInService.signOut()).called(1);
-    });
   });
 
   testWidgets('sign in functions call notify listener', (WidgetTester tester) async {
     final MockGoogleSignInPlugin mockSignInPlugin = MockGoogleSignInPlugin();
     when(mockSignInPlugin.onCurrentUserChanged).thenAnswer((_) => Stream<GoogleSignInAccount>.value(null));
     final GoogleSignInService signInService = GoogleSignInService(googleSignIn: mockSignInPlugin);
-    final FlutterBuildState buildState = FlutterBuildState(authServiceValue: signInService);
+
+    final FlutterBuildState buildState = FlutterBuildState(
+      cocoonService: MockCocoonService(),
+      authService: signInService,
+    );
 
     int callCount = 0;
-    buildState.addListener(() => callCount++);
+    buildState.addListener(() => callCount += 1);
 
-    // notify listener is called during construction of the state
     await tester.pump(const Duration(seconds: 5));
     expect(callCount, 1);
 
-    await buildState.signIn();
+    await signInService.signIn();
```

Comment:
Q: Throughout the code there is reference to `authService` and `signInService` (I did this, not you 😅). Have you found a preference for one of the two names? if so, we should consider switching to use just one name throughout the codebase.

Reply:
i have no preference one way or the other. Agreed that we should settle on one.

Reply:
added a todo